e2etests/testdata/stable/square_3d/elk/sketch.exp.svg

```diff
@@ -3,7 +3,7 @@
 id="d2-svg"
 style="background: white;"
 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
-width="390" height="599" viewBox="-90 -105 390 599"><style type="text/css">
+width="339" height="565" viewBox="-90 -90 339 565"><style type="text/css">
```

Comment:
3d went away

Reply:
Ah looks like we had a bunch of style keywords not under `style`. I just made that an error.